File: basic/01-max/01-max.c
#include <stdio.h>

int find_max(int* arr, int size) {
  int max = arr[0];
  for (int i = 1; i < size; i++) {
    if (arr[i] > max) {
      max = arr[i];
    }
  }
  return max;
}

int main() {
  int arr[] = {1, 2, 3, 4, 5};
  int size = sizeof(arr) / sizeof(arr[0]);
  int max_value = find_max(arr, size);
  return max_value;
}

File: basic/sorting/max/01-max.java
public class MaxValue {
  public static int findMax(int[] arr) {
    int max = arr[0]
    for (int i = 1; i < arr.length; i++) {
      if (arr[i] > max) {
        max = arr[i];
      }
    }
    return max;
  }

  public static void main(String[] args) {
    int[] arr = {1, 2, 3, 4, 5};
    int maxValue = findMax(arr);
    System.out.println(maxValue);
  }
}
